src/components/common/Video.js

```diff
@@ -108,7 +125,11 @@ export default class VideoComponent extends Component {
             }}
             resizeMode={Video.RESIZE_MODE_CONTAIN}
             source={this.props.video}
-            style={styles.video}
+            style={{
+              backgroundColor: 'transparent',
```

Comment:
Made background color of video transparent so that the underlying component can set the background if the video is to small